Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1412,6 +1419,18 @@ def lhegensim2018(fragment,howMuch):
     global step1LHEGenSimUp2018Default
     return merge([{'cfg':fragment},howMuch,step1LHEGenSimUp2018Default])
 
+def lhegensim2018RD(fragment,howMuch):
+    global step1Up2018Defaults
+    return merge([{'cfg':fragment},howMuch,{'--customise_commands': "\"process.source.numberEventsInLuminosityBlock=cms.untracked.uint32(1000) \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) ) \""},step1Up2018Defaults])
+#
+#* EcalLaserAPDPNRatios_Run_Dep_MC          (9IOVs)   run numbers: 1, 316082, 316720, 317527, ...
+#* EcalLaserAPDPNRatios_Run_Dep_MC_first_IOV   (1 IOV)   ---> used in SIM?
+
+# for run dependent MC (1000 events per LS), 2*1000 total events
+steps['TTbar_13UP18_RD']=lhegensim2018RD('TTbar_13TeV_TuneCUETP8M1_cfi',Kby(2,50))
```

Comment:
let's do 9k events

Reply:
Hmm. I think the idea we discussed is to have one workflow with small nEvents for short tests and one for RelVal production. For short tests we need 2 lumiblocks 5 events each with different IOVs so 2k seems too big.

Currently, you have 2k events which will turn to 200 lumiblocks and 9k events with 180 lumiblocks....

About name convention, I would call 9k events (normal relval) without "big" and add "_test" for smaller workflow everywhere e.g. lhegensim2018RD_test.

Reply:
I don't understand where the exact number of events is defined with runthematrix stuff.
We have a 

                 '-n'            : 10,

that defines 10 events, and that is what I was toldyesterday. Here a "2" that defines 2k events ...
I'm happy with any proposal, a tiny sample (10 events) for just tests and a big sample for physics perfomance, but I do we define this?

Reply:
Hi @amassiro , sorry for some late response. I think I have understood a bit more how the PR tests work (I'm also quite new in this career). The PR tests take number in -n 10 to run so in order to test number of lumiblocks, Chris & Fabio suggested you to have one small workflow (250202.183) with numberEventsInLuminosityBlock=5, your setup of 2k event might be OK because we will never submit this workflow as RelVal. But for real RelVal production, we will inject 9k events instead with proper lumi blocks and run numbers which are setup under 250202.184. Anyway, my comment about changing the names still stands. (About name convention, I would call 9k events (normal relval) without "big" and add "_test" for smaller workflow everywhere e.g. lhegensim2018RD_test.)

Btw, your current job 250202.183 failed at RECO step, please check log file here and fix it. https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-0c4e95/3560/runTheMatrix-results/250202.183_TTbar_13UP18_RD+TTbar_13UP18_RD+DIGIPRMXUP18_PU25_RD+RECOPRMXUP18_PU25_RD+HARVESTUP18_PU25/